api/infrastructure/parallelcluster-api.yaml

```diff
@@ -95,6 +95,11 @@ Parameters:
     Type: String
     Default: ''
 
+  IAMRoleAndPolicyPrefix:
+    Type: String
+    Description: 'Prefix for IAM roles and policies'
+    Default: ''
```

Comment:
We should add a constraint for this parameter to limit the maximum number of characters.

See https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html